src/main/java/sirius/web/http/LangHelper.java

```diff
@@ -46,7 +46,7 @@ private LangHelper() {
             if (m.matches()) {
                 double q = Value.of(m.group(4)).asDouble(1.0d);
```

Comment:
rename q into something else?

Reply:
I thought of that as well, but I also don't have a better name for it. q is part of the ACCEPT_LANGUAGE_PATTERN `" *([a-z]{2})(-[a-z]{2})? *(;q=([0-9.]+) *)?"` and I'm not exactly sure what it does.
If anyone has a good suggestion for the renaming, I can change it. Otherwise I would leave it as is

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language
The docs suggest that an appropriate name could be `quality` or `weight`